sidebar.c

```diff
@@ -113,6 +113,7 @@ enum SidebarSrc
  * |:--------|:--------------------------------------------------------
  * | \%B     | Name of the mailbox
  * | \%d     | Number of deleted messages
+ * | \%D     | Description of the mailbox
```

Comment:
Please can you add this to the main docs: [doc/manual.xml.head](https://github.com/neomutt/neomutt/blob/52828751bf608797a9bd3411c5896032a6d0e3bf/doc/manual.xml.head#L727-L828) line 727.
For extra points, please could you sort the options in the docs into alphabetical order :-)